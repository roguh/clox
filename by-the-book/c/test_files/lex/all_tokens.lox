( ) { } [ ] , . - + ; # & | ^ ~ ! = > / % * ** < != == >= <= << >>
id123 id_123 _id ID
"abc"
1234
0x2000deadbeef
and class else false for fun if nil or print return super this true var while
