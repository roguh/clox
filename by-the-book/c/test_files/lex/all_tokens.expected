   1:1    '('
   1:3    ')'
   1:5    '{'
   1:7    '}'
   1:9    '['
   1:11   ']'
   1:13   ','
   1:15   '.'
   1:17   '-'
   1:19   '+'
   1:21   ';'
   1:23   '#'
   1:25   '&'
   1:27   '|'
   1:29   '^'
   1:31   '~'
   1:33   '!'
   1:35   '='
   1:37   '>'
   1:39   '/'
   1:41   '%'
   1:43   '*'
   1:46   '**'
   1:48   '<'
   1:51   '!='
   1:54   '=='
   1:57   '>='
   1:60   '<='
   1:63   '<<'
   1:66   '>>'
   2:5    'id123'
   2:12   'id_123'
   2:16   '_id'
   2:19   'ID'
   3:5    '"abc"'
   4:4    1234
   5:14   0x2000deadbeef
   6:3    and
   6:9    class
   6:14   else
   6:20   false
   6:24   for
   6:28   fun
   6:31   if
   6:35   nil
   6:38   or
   6:44   print
   6:51   return
   6:57   super
   6:62   this
   6:67   true
   6:71   var
   6:77   while
   6:77   EOF
